enterprise/backend/test/metabase_enterprise/data_editing/api_test.clj

```diff
@@ -38,7 +39,7 @@
 
 (deftest table-operations-test
   (mt/with-premium-features #{:table-data-editing}
-    (mt/with-empty-h2-app-db
+    (mt/test-drivers #{:h2 :postgres}
```

Comment:
Can we rather introduce a driver feature, so we don't need to hard code this pair for every test?